docs/ops/state/large_state_tuning.md

```diff
@@ -142,6 +142,17 @@ by default. To enable this feature, users can instantiate a `RocksDBStateBackend
         new RocksDBStateBackend(filebackend, true);
 {% endhighlight %}
 
+**RocksDB Timers**
+
+For RocksDB, user can chose whether timers are stored on the heap (default) or inside RocksDB. Heap-based timers can have a better performance for smaller amounts of
```

Comment:
"a user"

Reply:
Please change "amounts" to "numbers" in both places in this sentence. (This is similar to the distinction in English between less and fewer. Continuous quantities have amounts, while countable quantities have numbers.)